documentation/src/main/asciidoc/ch02.asciidoc

```diff
@@ -687,6 +687,10 @@ With one exception also these constraints apply to the field/property level, onl
 	Supported data types::: `CharSequence`
 	Hibernate metadata impact::: Column length will be set to max
 
+`@CodePointLength(min=, max=)`:: Validates that code point length of the annotated character sequence is between `min` and `max` included
+	Supported data types::: `CharSequence`
+	Hibernate metadata impact::: Column length will be set to max
```

Comment:
Impact should probably be None, @gunnarmorling @gsmet ?

Reply:
Yes, good catch. I don't think we can safely set the length in the database.

Reply:
ok! I will change to "None".